src/extensions/validation/essayLimitByCharacter.js

```diff
@@ -294,6 +296,7 @@ function checkLimit(questionInstance, setUI = true) {
 function setValidationUI(questionInstance, isValid, strLength) {
     const id = questionInstance.getQuestion().response_id;
     const elContainer = document.getElementById(id);
+    if (!elContainer) return;
```

Comment:
What was the scenario where `checkLimit()` is called (that is the method that calls `setValidationUI()`) where you don't have a question on the page?

Reply:
I still investigate why document cannot find out the element by id, but y can try around with lazy loading configuration applied